java/cloud/unum/usearch/Index.java

```diff
@@ -375,9 +375,9 @@ public Config expansion_search(long _expansion_search) {
     } catch (UnsatisfiedLinkError e) {
       try {
         if (System.getProperty("os.name").equals("Mac OS X")) {
```

Comment:
Do we know what this property is equal to on Windows? We should probably link to .dll there.